next-desolates-map/pages/index.tsx

```diff
@@ -13,9 +14,13 @@ const Home: NextPage = (spaceRenderer) => {
 
     return (
         <>
+            <Head>
+                <title>DESOLATEs Map</title>
+                <meta name="description" content="3D map of all the planets in the DESOLATEs metaverse" />
+            </Head>
             <h1
                 className="absolute z-10 text-3xl tracking-tighter text-white select-none top-6 backdrop-blur backdrop-filter"
-                style={{ fontFamily: "Zen Dots" }}
+                style={{ fontFamily: "Nasalization Rg" }}
```

Comment:
We can remove this entire line, and instead add the class "font-nasalization" to `className=" ... "` so that we get 

`className="absolute z-10 ... backdrop-filter font-nasalization"`